pkg/loop/loopinstall/README.md

```diff
@@ -0,0 +1,75 @@
+# loopinstall
+
+A tool for installing Chainlink LOOP plugins from YAML configuration files.
+
+## Installation
+
+To install from source:
+
+```bash
+go install github.com/smartcontractkit/chainlink-common/pkg/loop/loopinstall@latest
+```
+
+Or clone and install locally:
+
+```bash
+git clone https://github.com/smartcontractkit/chainlink-common.git
+cd chainlink-common
+go install ./pkg/loop/loopinstall
+```
+
+## Usage
+
+```bash
+# Run `loopinstall --help` to see the options.
+loopinstall [options] <plugin-config-file> [<plugin-config-file>...]
+```
+
+## Configuration
+
+_(See `plugins.example.yaml` for a complete example configuration.)_
+
+Example configuration file structure:
+
+```yaml
+defaults:
+  goflags: "-ldflags='-s -w'" # Default Go build flags
+  goprivate: "github.com/myorg/myrepo,github.com/private-org/*" # Comma-separated list of private repo patterns
+
+plugins:
+  cosmos:
+    - name: "default"
+      enabled: true
+      moduleURI: "github.com/smartcontractkit/chainlink-cosmos"
+      gitRef: "f740e9ae54e79762991bdaf8ad6b50363261c056"
```

Comment:
nit: hash (to be consistent with Nix :)

Reply:
Technically git ref is more accurate :) I imagine we'd only want to install from hashes for releases (with a comment to indicate the `v...` version), so maybe on repos that use this for distributable releases we would fail if it's pointed to an e.g. branch? I think locally it might be nice just to use a branch though.